cpp/server/ct-mirror.cc

```diff
@@ -74,8 +74,8 @@ DEFINE_int32(log_stats_frequency_seconds, 3600,
              "Must be greater than 0.");
 DEFINE_int32(target_poll_frequency_seconds, 10,
              "How often should the target log be polled for updates.");
-DEFINE_string(etcd_host, "", "Hostname of the etcd server");
-DEFINE_int32(etcd_port, 0, "Port of the etcd server.");
+DEFINE_string(etcd_hosts, "",
```

Comment:
Vague feeling that this isn't just hosts... `etcd_servers`? If you do change it, make it the same everywhere, of course.